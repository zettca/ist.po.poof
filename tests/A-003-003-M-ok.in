2
qqqqqqqq
0
