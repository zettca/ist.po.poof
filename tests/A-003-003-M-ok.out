Menu Principal
1 - Novo
2 - Abrir
0 - Sair
Escolha uma opção: Ficheiro a abrir: O ficheiro não existe.
Menu Principal
1 - Novo
2 - Abrir
0 - Sair
Escolha uma opção: 
